src/test/cursor.rs

```diff
@@ -0,0 +1,86 @@
+use std::time::Duration;
+
+use bson::doc;
+use futures::{future::Either, StreamExt};
+
+use crate::{
+    options::{CreateCollectionOptions, CursorType, FindOptions},
+    test::{TestClient, LOCK},
+    RUNTIME,
+};
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+#[function_name::named]
+async fn tailable_cursor() {
+    let _guard = LOCK.run_concurrently().await;
+
+    let client = TestClient::new().await;
```

Comment:
is it worth adding an `init_db_and_coll_with_options` for this sequence for future tests?

Reply:
We could, I think I'd prefer a name that uses the word `create` somewhere in it, since explicitly creating the collection isn't something that `init_db_and_coll` does right now.

Reply:
Yeah that sounds fine, do we want to make that change or just wait until a second case needs it?

Reply:
I don't have a strong opinion either for or against doing it now, so I'll defer to whichever you prefer

Reply:
We're talking about it now, might as well do it while it's something we're thinking about.

Reply:
Added